templates/createreport.html

```diff
@@ -20,6 +20,9 @@ <h1>This week at a glance</h1>
     .Undertime {
         color: #DC3220;
     }
+    .bold {
+        font-weight: bold
+    }
```

Comment:
Please use the pre-defined class from Bootstrap 5, no need to define our own: https://getbootstrap.com/docs/5.0/utilities/text/#font-weight-and-italics